app/src/androidTest/java/net/skyscanner/backpack/bottomsheet/BpkBottomSheetTest.kt

```diff
@@ -66,26 +68,24 @@ class BpkBottomSheetTest : BpkSnapshotTest() {
   }
 
   @Test
+  @Variants(BpkTestVariant.Default)
   fun expanded() {
-    assumeVariant(BpkTestVariant.Default)
-    val asyncScreenshot = prepareForAsyncTest()
-
-    rule.scenario.onActivity { activity ->
-      val root = setupBottomSheet()
-      activity.setContentView(root)
-      root.post {
-        bottomSheetBehaviour.state = STATE_EXPANDED
+    rule.scenario.waitForActivity().also { activity ->
```

Comment:
Have you noticed any weird behaviour with that method? Is there any cases, which this method doesn't cover anymore?

Reply:
No weird behaviour I could see - mainly need to avoid taking the snapshot in any lambda etc (that's what the `prepareForAsync` was there before to get around that)